PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+#### Story / Bug id:
+123 [link to the issue]
+
+#### Description:
+Describe changes made in your pull request here
+
+#### Migrations:
+N/A
+
+#### Rollback:
+N/A
+
+#### New imports / dependencies:
+N/A
+
+#### Unit Tests:
+N/A
```

Comment:
What does it mean? Should I list new unit tests here? Or describe what I (potentially) added to the testing suite?

Reply:
When you don't have a habit of writing tests this could be actually self-reflective. "Oh so... I wrote this function so what kind of tests do I need here? Since it is expected to list sth...".

Also this could be early checker for complexity od the code. If you have a lot of those the chance is that you like spaghetti, aren't you? :D

Reply:
Hard-copy from other project. :D Could be removed.